.github/workflows/release.yml

```diff
@@ -122,82 +122,105 @@ jobs:
-        aws_region: ${{fromJson(github.event.inputs.aws_region).aws_region}}
-        # We need to define the raw parameters to the matrix here but will customize each
-        # below.
-        name:
-          - java-awssdk-agent
-          - java-awssdk-wrapper
-          - java-okhttp-wrapper
-          - nodejs-awssdk
-          - python38
-          - dotnet-awssdk-wrapper
-        include:
-          - name: java-awssdk-agent
-            layer_kind: java-agent
-            language: java
-            build_directory: java
-            build_command: ./build.sh
-            terraform_directory: sample-apps/java-agent-aws-sdk-terraform
-            amp_regions: [ "us-west-2", "us-east-1", "us-east-2", "eu-central-1", "eu-west-1"]
-            expected_trace_template: adot/utils/expected-templates/java-awssdk-agent.json
-            expected_metric_template: adot/utils/expected-templates/java-awssdk-agent-metric.json
-          - name: java-awssdk-wrapper
-            layer_kind: java-wrapper
-            language: java
-            build_directory: java
-            build_command: ./build.sh
-            terraform_directory: sample-apps/java-wrapper-aws-sdk-terraform
-            expected_trace_template: adot/utils/expected-templates/java-awssdk-wrapper.json
-          - name: java-okhttp-wrapper
-            layer_kind: java-wrapper
-            language: java
-            build_directory: java
-            build_command: ./build.sh
-            terraform_directory: sample-apps/java-wrapper-okhttp-terraform
-            expected_trace_template: adot/utils/expected-templates/java-okhttp-wrapper.json
-          - name: nodejs-awssdk
-            layer_kind: nodejs
-            language: nodejs
-            build_directory: opentelemetry-lambda/nodejs
-            build_command: npm install
-            terraform_directory: sample-apps/nodejs-aws-sdk-terraform
-            expected_trace_template: adot/utils/expected-templates/nodejs-awssdk.json
-          - name: python38
-            layer_kind: python38
-            language: python
-            build_directory: opentelemetry-lambda/python
-            build_command: |
-              cd sample-apps
-              ./build.sh
-            terraform_directory: sample-apps/python-aws-sdk-aiohttp-terraform
-            expected_trace_template: adot/utils/expected-templates/python.json
-          - name: dotnet-awssdk-wrapper
-            layer_kind: collector
-            language: dotnet
-            build_directory: dotnet
-            build_command: ./build.sh
-            terraform_directory: sample-apps/dotnet-wrapper-aws-sdk-terraform
-            expected_trace_template: adot/utils/expected-templates/dotnet-awssdk-wrapper.json
+    name: Set Smoke Test Configuration Values
+    outputs:
+      layer_kind: ${{ steps.set-smoke-test-outputs.layer_kind }}
+      language: ${{ steps.set-smoke-test-outputs.language }}
+      build_directory: ${{ steps.set-smoke-test-outputs.build_directory }}
```

Comment:
Does that mean in release step we have to input more args?+

Reply:
Hm I don't think so! The arguments are the same for now, the only caveat is that this code doesn't allow for more than 1 Soak Test. For example Java Wrapper seems to do 2 Soak Tests after the release.

Is there any additional input you were thinking we might need with this change?

Reply:
It is ok that run only one smoke test in releasing since every time we just release one layer kind.

Reply:
Yeah I'd agree that only 1 test per layer is fine - we run the canary tests on all sample apps anyway, the smoke tests should just be a quick confirmation that nothing obvious is broken.

Reply:
The one exception is possibly for the collector, where we'll eventually want to run both Go and .NET smoke tests for the collector. Not too urgent at the moment, but just another consideration we can discuss with team

Reply:
yes, a layer may be used in multiple sample apps, such as java-wrapper and collector.